### ISSUE TITLE
Add slider input to SVG slide

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#229]
```
Marcin's sample has that already
```

Original issue reported on code.google.com by `erne...@chromium.org` on 6 Oct 2010 at 7:13

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Apr 2011 at 4:49
- Added labels: **Type-Feature**, **Priority-P3**, **Milestone-6**
- Removed labels: **Type-Enhancement**, **Priority-Medium**, **Milestone-4**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Apr 2011 at 4:49

**Comment by GoogleCodeExporter** [comment]
Original comment by `han...@google.com` on 8 Dec 2011 at 12:30
- Added labels: **Milestone-Q42011-1**
- Removed labels: **Milestone-6**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision 7906095f0061
```

Original comment by `han...@google.com` on 9 Dec 2011 at 11:06
- Changed state: **Fixed**
